WcaOnRails/app/views/static_pages/about.html.erb

```diff
@@ -24,14 +24,13 @@
 
   <p><%= t('about.structure.committees') %>
   <ul>
-    <li><%= t('about.structure.wdc.name_html') %>: <%= format_team_members('wdc') %> <br />
-      <%= t('about.structure.wdc.description') %></li>
-    <li><%= t('about.structure.wrc.name_html') %>: <%= format_team_members('wrc') %> <br />
-      <%= t('about.structure.wrc.description') %></li>
-    <li><%= t('about.structure.results.name_html') %>: <%= format_team_members('results') %> <br />
-      <%= t('about.structure.results.description') %></li>
-    <li><%= t('about.structure.software.name_html') %>: <%= format_team_members('software') %> <br />
-      <%= t('about.structure.software.description') %></li>
+    <% [ "wct", "wdc", "wrc", "results", "software" ].each do |friendly_id| %>
+      <% team = Team.find_by_friendly_id!(friendly_id) %>
```

Comment:
The explicit list is perfect as long as a team is not added, removed or renamed.

I'd handle that by a test that asserts that `Team.all` is equivalent to `[ "wct", "wdc", "wrc", "results", "software" ]`.

That list should probably be on the `Team` class. `Team.all_friendy_ids`.